### PR TITLE
update documentation for offeringCollaborationType in SpaceTemplateInfo guide

### PR DESCRIPTION
**Pull request by bhumit**
@a-kent   please review

### CODE REVIEW

File: guides/V1_space_template_main.md [hunk 1]
```diff
@@ -18,4 +18,4 @@ as well as a set of required apps that are automatically added to the space on c
...
 | teamId |String |The id of the team in which the space will be created, derived either from the offering with which the template is associated (in the case of an offering template) or from the user who created the template (in the case of a user-created custom template)|
-| offeringCollaborationType |OfferingCollaborationType |The type of offering with which the space will be associated, derived either from the offering with which the template is associated (in the case of an offering template) or from the user who created the template (in the case of a user-created custom template)|
+| offeringCollaborationType |OfferingCollaborationType |An offering defines the capabilities granted to users with a subscription to a given product. There are two broad types of offering: PERSONAL and TEAM. A user with a subscription to a TEAM offering belongs to a team. The type of offering with which the space will be associated, derived either from the offering with which the template is associated (in the case of an offering template) or from the user who created the template (in the case of a user-created custom template)|
```
Comment (a-kent) [inline guides/V1_space_template_main.md]: I would swap this around and add the new explanation of an offering after the existing text.

| property      | type          | description  |
| ------------- |:------------- |:-----|
| offeringCollaborationType |OfferingCollaborationType | The type of offering with which the space will be associated, derived either from the offering with which the template is associated (in the case of an offering template) or from the user who created the template (in the case of a user-created custom template). An offering defines the capabilities granted to users with a subscription to a given product. The two broad types of offering are `PERSONAL` and `TEAM`. A user with a subscription to a `TEAM` offering belongs to a team. |

**Comment by a-kent** [comment]
Thanks for the update. :+1:

**Comment by bhumit** [comment]
@jarusso / @miguelestrada Can you please review / merge if acceptable?

**Comment by jarusso** [comment]
looks good to me - @miguelestrada ?

**Review by miguelestrada — APPROVED** [review]
The topics of offering types and subscriptions are brought up here without much context.  We will separately document the separation of concerns between: Work Services APIs,  Offerings and Products,  Subscriptions.   @jarusso will set up time for this.

**Pull request merged**